k8s/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: minio-kv-deployment
+  labels:
+    app: minio-kv
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: minio-kv
+  template:
+    metadata:
+      labels:
+        app: minio-kv
+    spec:
+      containers:
+      - name: minio-kv
+        image: alexellis2/minio-kv:0.1.0
+        env:
+        - name: MINIO_ACCESS_KEY
```

Comment:
Hi, the env-vars need to be brought in via secrets.

See this example https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables

cc @utsavanand2

Reply:
Sure, moved to a Secret. Those values, even base64 encoded, should probably not be in git, so they are here more as an example, like in the README.